Packs/UnifiVideoNVR/Integrations/UnifiVideo/UnifiVideo.yml

```diff
@@ -44,6 +44,12 @@ configuration:
   name: fetch_time
   required: false
   type: 0
+- defaultvalue: '0'
```

Comment:
why did you set the `defaultvalue` to 0?

Reply:
0 is unlimited.